src/WorkBC.Data/Migrations/20240413000723_InsertNocCodes2021.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System.Text.Json;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System;
+using WorkBC.Data.Model.JobBoard;
+using Microsoft.EntityFrameworkCore.Metadata.Internal;
+using WorkBC.Data.Model.Enterprise;
```

Comment:
Do we need this import?

Reply:
Removed the unnecessary using commands in the latest commit.